bin/build_fixtures

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# A script to quickly build all of the .pptx fixtures from the project's root
+
+cd ./spec/fixtures/
+for f in ./*; do
+  if [[ -d $f ]]; then
+    ( cd $f && echo "Building $f.pptx" && zip -r ../$f.pptx ./* > /dev/null )
```

Comment:
🚨 We've found that PowerPoint, Word, Excel, etc have an allergic reaction to however `zip` compresses a package. That's why Amos and I wrote `bin/zipflip`:  The Ruby gem `rubyzip` _doesn't_ cause the same problems that the `zip` utility does.